lib/symbol-daily.types.ts

```diff
@@ -0,0 +1,16 @@
+/** String in the form "YYYY-MM-DD" */
+type ApiDate = string;
+export interface SymbolDailyAlphavantageResponse {
+  "Time Series (Daily)": {
+    [date: ApiDate]: {
+      "1. open": string;
+      "2. high": string;
+      "3. low": string;
+      "4. close": string;
+      "5. volume": string;
+    };
+  };
+}
```

Comment:
I would move this chunk to `app/api/stock/[symbol]/daily/types.ts` to clarify that the client code shouldn't need to access it

Reply:
Done.